Bottom: 73bf01b96a94bdfa8cfd3965c502a70d13ed6182
Top:    5d02fb65d212782720f5d28fc3a2c87621ede9d6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-02-22 00:21:03 +0000

Change scroll-wheel bindings to match more GTK applications









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 0dc18a1..c36d23c 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
